### PR TITLE
add support for Radxa ROCK 3C, Radxa ZERO 3E and 3W

### PR DESCRIPTION
**Pull request by RadxaNaoki**
please read commit messages for the detail.

### CODE REVIEW

**Comment by RadxaNaoki** [comment]
it seems I did something wrong in target/linux/rockchip/patches-6.6/047-*. I'll fix it...

File: package/boot/uboot-rockchip/patches/000-backport-upstream-dts-sync.patch
Comment (maurerr) [inline package/boot/uboot-rockchip/patches/000-backport-upstream-dts-sync.patch:1]: why sync other dts files except zero3e/w+rock3c?

**Review by maurerr — APPROVED** [review]

**Comment by maurerr** [comment]
cc @1715173329

**Comment by RadxaNaoki** [comment]
> The first commit title should start with `uboot-rockchip`, otherwise LGTM.
> 
> EDIT: please also refresh uboot patches with `make package/uboot-rockchip/refresh V=s`

done

thanks!

**Comment by RadxaNaoki** [comment]
conflict with rk3588 PRs...

**Comment by RadxaNaoki** [comment]
is there any issue to merge this PR?

**Comment by maurerr** [comment]
we need one of the maintainers to find time - maybe @hauke or @PolynomialDivision

**Comment by RadxaNaoki** [comment]
may I close this PR and migrate to https://github.com/openwrt/openwrt/pull/16185 ?

**Comment by hauke** [comment]
The CI found a problem in this PR:
```

Run . .github/workflows/scripts/ci_helpers.sh
target/linux/rockchip/patches-6.6/053-v6.9-arm64-dts-rockchip-Add-support-for-NanoPi-R6S.patch
Kernel patches for rockchip/armv8 require refresh. (run 'make target/linux/refresh' and force push this pr)
You can also check the provided artifacts with the refreshed patch from this CI run.
Error: Process completed with exit code 1.
```

**Comment by RadxaNaoki** [comment]
> The CI found a problem in this PR:
> 
> ```
> 
> Run . .github/workflows/scripts/ci_helpers.sh
> target/linux/rockchip/patches-6.6/053-v6.9-arm64-dts-rockchip-Add-support-for-NanoPi-R6S.patch
> Kernel patches for rockchip/armv8 require refresh. (run 'make target/linux/refresh' and force push this pr)
> You can also check the provided artifacts with the refreshed patch from this CI run.
> Error: Process completed with exit code 1.
> ```

I know. there was a patch for it. but I squashed it by request https://github.com/openwrt/openwrt/pull/16185#pullrequestreview-2256397821

**Pull request closed without merging**